app/(tabs)/two.tsx

```diff
@@ -61,12 +61,12 @@ export default function TabTwoScreen() {
         ", " +
         event.longitude +
         " from " +
-        dtf.format(event.startTime) +
+        dtf.format(event.startTime * 1000 /* fix */) +
```

Comment:
Why this fix is needed?

Reply:
https://stackoverflow.com/questions/4676195/why-do-i-need-to-multiply-unix-timestamps-by-1000-in-javascript
Backend uses seconds since 01.01.1970, JS uses milliseconds